index.html

```diff
@@ -1,86 +1,96 @@
-      </div>
-      <div id="posts">
-        <h1>Posts</h1>
-      </div>
-      <div id="about">
-        <h1>About</h1>
-      </div>
-    </div>
-    <div id="footer">
-      <div class="posts-container footer">
-        <div class="social-icon">
-            <h3 class="title">Social</h3>
-            <ul class="list-unstyled icon">
-                <li><a href=""><i class="fa fa-facebook"></i></a></li>
-                <li><a href=""><i class="fa fa-twitter"></i></a></li>
-                <li><a href=""><i class="fa fa-instagram"></i></a></li>
-                <li><a href=""><i class="fa fa-youtube-play"></i></a></li>
-            </ul>
-        </div>
-        <div class="categories">
-            <h3 class="title">Categories</h3>
-            <div class="contain-box">
-                <a href="#" class="box">Custom</a>
-                <a href="#" class="box">Me</a>
-                <a href="#" class="box">Contact</a>
-                <a href="#" class="box">Custom</a>
-                <a href="#" class="box">Me</a>
-                <a href="#" class="box">Contact</a>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <main>
+        <section class="postSection">
+            <div class="container postContain">
+                <div class="txtTitle">
+                    <span>All stories</span>
+                    <div class="arrowStories">
+                        <img src="image/Arrow 1.png" alt="ArrowStories">
+                    </div>
+                </div>
+                <div class="post">
+                    <div class="postImg">
+                        <img src="image/post_1.png" alt="postImg">
+                    </div>
+                    <div class="postMain">
+                        <a href="" class="postCategory">Chuyện của tôi</a>
+                        <a href="" class="postTitle">Click this card for redirect to full story</a>
+                        <div class="postDescription">Lorem ipsum dolor sit amet consectetur adipiscing elit Lorem ipsum dolor sit amet consectetur adipiscing elit. ipsum dolor sit amet consectetur adipiscing elit.</div>
+                        <div class="postInfoShare">
+                            <p class="postInfo">bởi <a href="" style="font-weight: 700;">Pikachu</a>, 07/07/2011</p>
+                            <a href="" class="postShare">Share</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="post">
+                    <div class="postImg">
+                        <img src="image/post_2.png" alt="postImg">
+                    </div>
+                    <div class="postMain">
```

Comment:
Read carefully our naming convention for HTML, CSS vs. JavaScript files: https://www.dropbox.com/scl/fi/k4a5rv4kcgt234z6no0ih/-Coding-Convention.paper?dl=0&rlkey=ka3xx7whfsjq61scx8smcesya
In this case, HTML and CSS file **must be write lowercase and when it have more than 1 word must separate each other by using “ - “**.

Reply:
Fix other naming convention errors in all of your files.

Reply:
@nguyenvl179 You haven't resolved this review.